### PR TITLE
fix(runtime-wry): window edge not working after setting resizable false and decorated false dynamically

### PR DESCRIPTION
**Pull request by Legend-Master**
Not sure if it's related with #10205

### CODE REVIEW

**Comment by Legend-Master** [comment]
I'm not sure if we also want to disable it on fullscreen and maximize or is it something the user should do manually

**Comment by FabianLars** [comment]
> I'm not sure if we also want to disable it on fullscreen and maximize or is it something the user should do manually

I think we used to do that 🤔

**Comment by Legend-Master** [comment]
Looking at the code it seems to be handled by amr's code already, ~~didn't test it though~~ seems like it's working

File: .changes/resize-false-undecorated-edge.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"tauri-runtime-wry": "patch:bug"
+---
+
+Fix window edge not working after setting resizable false and decorated false dynamically
```
Comment (amrbashir) [inline .changes/resize-false-undecorated-edge.md:5]: I am not sure I understand the problem, if decoration is false and resizable is false, window edge should not work

Comment (Legend-Master) [reply .changes/resize-false-undecorated-edge.md:5]: Shouldn't the edges of an undercoated window with resize disabled receive events and being clickable?

Comment (amrbashir) [reply .changes/resize-false-undecorated-edge.md:5]: yeah it should, is it not receiving these events?

Comment (Legend-Master) [reply .changes/resize-false-undecorated-edge.md:5]: no, that part is blocked by the top level resize window and not interactable at all

Comment (amrbashir) [reply .changes/resize-false-undecorated-edge.md:5]: ah I see where the bug is, I will push a better fix

Comment (amrbashir) [reply .changes/resize-false-undecorated-edge.md:5]: @Legend-Master could you give 14af33bc765751b07de187884995c8066902a697 a test?

Comment (Legend-Master) [reply .changes/resize-false-undecorated-edge.md:5]: Doesn't seem to work, after setting both resize and decoration to false, the entire window becomes not interactable

Comment (amrbashir) [reply .changes/resize-false-undecorated-edge.md:5]: I retested and seemed like I can click just fine, outside and inside the resize region.

Comment (amrbashir) [reply .changes/resize-false-undecorated-edge.md:5]: nvm, I was mislead by the hover effect, I reverted my changes

**Review by amrbashir — APPROVED** [review]

File: .changes/resize-false-undecorated-edge.md (hunk 1 above)
Comment (amrbashir) [reply .changes/resize-false-undecorated-edge.md:5]: kinda weird that `HTTRANSPARENT` didn't work, will need to research it more I guess. Your solution is fine for now.

**Pull request merged**